install/firmware.csv

```diff
@@ -12,14 +12,16 @@ Campbell Scientific,CR6,04111,CR6.Std.11.01,2021-11-17T14:10:00Z,9999-01-01T00:0
 Campbell Scientific,CR6,04112,CR6.Std.xx.xx,1901-01-01T00:00:00Z,9999-01-01T00:00:00Z,
 Campbell Scientific,CR6,04114,CR6.Std.11.01,2021-10-14T00:00:00Z,9999-01-01T00:00:00Z,
 Campbell Scientific,CR6,04115,CR6.Std.04,2020-10-24T00:00:00Z,2022-01-26T23:59:00Z,
-Campbell Scientific,CR6,04115,CR6.Std.11.01,2022-01-27T00:00:00Z,9999-01-01T00:00:00Z,
+Campbell Scientific,CR6,04115,CR6.Std.11.01,2022-01-27T00:00:00Z,2023-11-08T23:59:00Z,
+Campbell Scientific,CR6,04115,CR6.Std.12.02,2023-11-09T00:00:00Z,9999-01-01T00:00:00Z,
```

Comment:
hi @ricardojohansson , thanks for this!
according to delta , the CR6 with sn. 04115 is not installed anywhere at the moment. 
In the `install/dataloggers.csv` file, the entries currently are:
```
Campbell Scientific,CR6,04115,Ruapehu Crater Lake Outlet,,2018-05-03T00:00:00Z,2022-01-17T22:39:59Z
Campbell Scientific,CR6,04115,Tongariro Lower Emerald Spring,,2022-03-30T21:10:00Z,2023-03-13T00:00:00Z
```

can you please confirm where this datalogger is installed? thanks!